bash/config.sh

```diff
@@ -103,6 +101,7 @@ function get_existence {
 }
 
 function import {
+  set -e
```

Comment:
Do you know the behavior of setting this in functions vs in scripts?

Reply:
Looking at the commit for that first linked build, looks like you didn't have the psql ON_ERROR_STOP=1 set properly
https://github.com/NYCPlanning/db-zoningtaxlots/commit/30b5f24c002830232ae3d55895ec202c06548287
So I think despite psql printing the error, it wasn't returning an error code to bash, so I think regardless that's expected behavior

Reply:
nice! I eventually got the psql variable setting right with https://github.com/NYCPlanning/db-zoningtaxlots/pull/132/commits/31ec6714c76f53a7fe35cc771b6a6e0096cd97bf